extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -152,7 +152,7 @@ public void initialize() {
             startAutomaticRefresh();
 
         } catch (Exception ex) {
-            String msg = "Error occurred while initializing NTP handler: " + ex.getMessage();
+            String msg = "Error occurred while initializing NTP handler: {}" + ex.getMessage();

```

Comment:
Err, that won't work here as this is just the String variable, not the logging line.